### PR TITLE
Responsive coc page

### LINKED ISSUE #132: Make code-of-conduct page responsive

### PR DESCRIPTION
**Pull request by shwetalsoni**
### Fixes #132

**Phone View**
<br/>
Screenshot (7)
<br/>
Screenshot (8)
<br/>
Screenshot (9)
<br/>
**Tablet View**
Screenshot (10)
<br/>
Screenshot (11)
<br/>
Screenshot (12)

### CODE REVIEW

**Review by salil-naik — CHANGES_REQUESTED** [review]
Hi. Please check your code. 6 backend files are edited in your PR.

**Comment by shwetalsoni** [comment]
@salil-naik  sir I didn't editted any of them but when i pulled the changes they were shown in my git tree . So i had to commit them to avoid any conflicts

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-import { BrowserRouter as Router, Switch } from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
 // css
 import "./css/main.scss";
 
 // components
 import Footer from "./components/Footer.jsx";
 import Header from "./components/Header.jsx";
+import Notification from"./components/Notification.jsx";
```
Comment (salil-naik) [inline frontend/src/App.js:10]: why have you called this here?

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+
+
 
 // pages
 import Home from "./pages/home.jsx";
 import Registration from "./pages/Registration";
-import AboutUs from "./pages/AboutUs.jsx";
-import AdminProfile from "./pages/adminProfile.jsx";
+import AboutUs from "./pages/AboutUs";
```
Comment (salil-naik) [inline frontend/src/App.js:17]: add .jsx

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
+import AdminProfile from "./pages/adminProfile";
```
Comment (salil-naik) [inline frontend/src/App.js:18]: add .jsx

File: frontend/src/css/settings/_global.scss [hunk 4]
```diff
@@ -1,11 +1,11 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
```
Comment (salil-naik) [inline frontend/src/css/settings/_global.scss:1]: Revert this file to as it was before.

**Review by salil-naik — CHANGES_REQUESTED** [review]

**Comment by salil-naik** [comment]
@shwetalsoni please attach mobile and tablet screenshots. 
@gauravdas014 please have a look at the backend files and let me know if it's okay to approve the PR.

**Review by gauravdas014 — CHANGES_REQUESTED** [review]
Might have forgotten to do a git pull. Merging the PR will break the backend code. Some files have been removed, and some have been edited in this pr. Details are as follows:
1. in app.js line 12 and 44 is removed, which should not be removed
2. swaggerDefinition.js file have been removed earlier, so this file should not be again marge them to master.
3. projectController.js should not be removed as it's being used in project section. Current PR is deleting the file
4. forumPostCommentModel should not be removed
5.authRoutes.js is updated, it was edited later on
6. Some lines are deleted in projectRoutes.js, that must not happen

**Comment by gauravdas014** [comment]
@salil-naik the pr needs a lot of changes. The local repo of @shwetalsoni was not updated

**Comment by shwetalsoni** [comment]
@gauravdas014 when i pulled the changes from upstream the files you mentioned being deleted or changed were already that way before. I didn't know why it was shown in git tree on being pulled. So i had to commit them. 

If you know how to fix this lemme know please.

**Comment by gauravdas014** [comment]
@shwetalsoni can you revert the commit which added all those unnecessary changes?

**Comment by shwetalsoni** [comment]
@gauravdas014  I ll do it  as soon i get free

**Comment by shwetalsoni** [comment]
@gauravdas014 I have reverted the files . Please check if everything is fine

**Comment by gauravdas014** [comment]
@shwetalsoni Just remove the swaggerDefinition.js file, rest looks fine to me.

**Comment by shwetalsoni** [comment]
@gauravdas014 Done
@salil-naik Please check if anymore changes are to be done

**Review by gauravdas014 — APPROVED** [review]

**Comment by gauravdas014** [comment]
@shwetalsoni looks good to me. Thanks!

**Comment by salil-naik** [comment]
Thanks a lot for your help @gauravdas014

**Pull request merged**